src/domain/app.ts

```diff
@@ -45,6 +45,6 @@ export const MempoolExplorerURLs: ExplorerURLs = {
 
 export const MempoolTestnetExplorerURLs: ExplorerURLs = {
   type: 'Mempool',
-  electrsURL: 'https://liquid.network/liquidtestnet',
-  esploraURL: 'https://liquid.network/liquidtestnet/api',
+  electrsURL: 'https://liquid.network/testnet',
+  esploraURL: 'https://liquid.network/testnet/api',
```

Comment:
```suggestion
  electrsURL: 'https://liquid.network/liquidtestnet/api',
  esploraURL: 'https://liquid.network/testnet',
```

It seems to me the elctrs backend API can be found at 

https://liquid.network/liquidtestnet/api/blocks

and frontend at

https://liquid.network/testnet/blocks